web/js/custom-script.js

```diff
@@ -0,0 +1,18 @@
+let noRouteParagraph = document.querySelector('p');
+
+let noRouteText = noRouteParagraph.innerHTML;
+
+let noRouteTextArray = noRouteText.split(' ');
+
+if (noRouteTextArray.length >= 3) {
+  let spaceText = noRouteTextArray[2];
+
+  if (spaceText === 'SPACE') {
+    let newNoRouteText = noRouteText.replace(
```

Comment:
honestly, I didn't have an idea of this change 😅

Reply:
oh, I already got the idea. pagebuilder doesn't allow wrapping a word to a custom span. yeah, could be